server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -216,11 +216,11 @@ public void insert(InsertRowPlan insertRowPlan) throws WriteProcessException {
     }
 
     // update start time of this memtable
-    tsFileResource.updateStartTime(insertRowPlan.getDeviceId(), insertRowPlan.getTime());
+    tsFileResource.updateStartTime(insertRowPlan.getDeviceId().getFullPath(), insertRowPlan.getTime());
```

Comment:
Same as above, if we change the key's type of the map to PartialPath, we don't have to call the getFullPath() method.